lib/locales/README.md

```diff
@@ -6,10 +6,10 @@ If you are using pagy with some language missing from the dictionary files, plea
 
 The `pagy.info.single_page`, `pagy.info.item_name` and `pagy.items` keys of the yaml file are pluralized according to the plural rules of the language.
 
-If you are going to submit a dictionary, please check wether the pluralization type of your language exists in the [plurals.rb](https://github.com/ddnexus/pagy/blob/master/lib/locales/plurals.rb) file. If it is missing, please, report it. Thanks.
+If you are going to submit a dictionary, please check wether the pluralization type of your language exists in the [lib/locales/utils/p11n.rb](utils/p11n.rb) file. If it is missing, please, report it. Thanks.
```

Comment:
I prefer using relative links in such cases.

Reply:
Yes. Thank you!